server/api/handlers/account_info.go

```diff
@@ -2,15 +2,37 @@ package handlers
 
 import (
 	"context"
+	"github.com/eoscanada/eos-go"
+	"platform-backend/models"
 	"platform-backend/server/api/ws_interface"
 )
 
+type PlayerInfoResponse struct {
+	Balance          eos.Asset         `json:"balance"`
+	ActivePermission eos.Authority     `json:"activePermission"`
+	OwnerPermission  eos.Authority     `json:"ownerPermission"`
+	LinkedCasinos    []*CasinoResponse `json:"linkedCasinos"`
+}
+
+func toPlayerInfoResponse(p *models.PlayerInfo) *PlayerInfoResponse {
+	ret := &PlayerInfoResponse{
+		Balance:          p.Balance,
+		ActivePermission: p.ActivePermission,
+		OwnerPermission:  p.OwnerPermission,
+	}
+	for _, casino := range p.LinkedCasinos {
```

Comment:
Size is known in advance; it is better to allocate memory of the right size immediately